r/README.md

```diff
@@ -37,15 +37,11 @@ Conda users on Linux and macOS can install `arrow` from conda-forge with
     conda install -c conda-forge r-arrow
 
 On macOS and Windows, installing a binary package from CRAN will handle
-Arrow’s C++ dependencies for you. On Linux, unless you use `conda`
-you’ll need to first install the C++ library. See the [Arrow project
-installation page](https://arrow.apache.org/install/) to find
-pre-compiled binary packages for some common Linux distributions,
-including Debian, Ubuntu, and CentOS. You’ll need to install
-`libparquet-dev` on Debian and Ubuntu, or `parquet-devel` on CentOS.
-This will also automatically install the Arrow C++ library as a
-dependency. Other Linux distributions must install the C++ library from
-source.
+Arrow’s C++ dependencies for you. On Linux, unless you use `conda`, the
```

Comment:
README.md is a generated file; these edits were made to the source README.Rmd in the last patch but I didn't regenerate README.md until now.